hudi-flink-datasource/hudi-flink/src/test/java/org/apache/hudi/sink/ITTestDataStreamWrite.java

```diff
@@ -234,6 +234,7 @@ private void testWriteToHoodie(
       Pipelines.clean(conf, pipeline);
       Pipelines.compact(conf, pipeline);
     }
+    Pipelines.cluster(conf, rowType, pipeline);
     JobClient client = execEnv.executeAsync(jobName);
```

Comment:
Revert this line and add a new test for INSERT mode and async clustering.

Reply:
Ok, i will add this later.

Reply:
Please remember to add it then ~